dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/LtsBomIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dashboard;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.eclipse.aether.artifact.Artifact;
+import com.google.cloud.tools.opensource.dependencies.Artifacts;
+import com.google.cloud.tools.opensource.dependencies.Bom;
+import com.google.cloud.tools.opensource.dependencies.MavenRepositoryException;
+import com.google.common.io.ByteStreams;
+
+class LtsBomIntegrationTest {
+  
+  // todo immutable
+  private static final Map<String, String> bomVersions = new HashMap<>();
+
+  public static void main(String[] args) 
+      throws MavenRepositoryException, IOException {
+    Path bomPath = Paths.get("..", "boms", "cloud-lts-bom", "pom.xml").toAbsolutePath();  
+    List<Artifact> managedDependencies = Bom.readBom(bomPath).getManagedDependencies();
+    
+    for (Artifact artifact : managedDependencies) {
+      bomVersions.put(Artifacts.makeKey(artifact), artifact.getVersion());
+    }
+    
+    for (Artifact artifact : managedDependencies) {
+      String testJar = buildTestJarUrl(artifact);
+      HttpURLConnection connection = (HttpURLConnection) new URL(testJar).openConnection();
+      connection.setRequestMethod("HEAD");
+      int result = connection.getResponseCode();
+      if (result == HttpURLConnection.HTTP_OK) {
+        testArtifact(artifact);
+      } else {
+        System.err.println(artifact + " does not have test jar");        
+      }
+    }
+  }
+
+  private static void testArtifact(Artifact artifact) throws IOException {
+    // download the pom.xml from Maven Central
+    URL remote = buildPomUrl(artifact);
+    Path temp = Files.createTempDirectory("pom");
+    Path local = temp.resolve("pom.xml");
+    Files.copy(remote.openStream(), local);
+    
+    // todo how to locate maven?
```

Comment:
Let the PATH environment variable take care of this. It should work in GitHub Actions and local computer.